.github/workflows/ci.yml

```diff
@@ -297,3 +297,54 @@ jobs:
           hub release create -m $tag_name $tag_name $(for f in cachepot-*.tar.gz*; do echo "-a $f"; done)
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+
+  deploy-gh-pages:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - name: Checkout local repo
+      uses: actions/checkout@v2
+
+    - name: Clone cachepot repo
```

Comment:
Do we need to double-clone the same repository? Can't we run `mdbook build docs/` from the current directory?